src/LinkMe/LinkMe.Domain/User.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Microsoft.AspNetCore.Identity;
+
+namespace LinkMe.Domain
+{
+    public class User : IdentityUser
+    {
+        public int Id { get; set; }
```

Comment:
To not forget to remove the ID property as I believe IdentityUser already introduces a primary key